google/cloud/bigtable/cluster.py

```diff
@@ -85,6 +89,28 @@ class Cluster(object):
                    :data:`google.cloud.bigtable.enums.Cluster.State.CREATING`.
                    :data:`google.cloud.bigtable.enums.Cluster.State.RESIZING`.
                    :data:`google.cloud.bigtable.enums.Cluster.State.DISABLED`.
+
+    :type min_serve_nodes: int
+    :param min_serve_nodes: (Optional) The minimum number of nodes in the cluster. Must be
```

Comment:
I would reword this slightly to point out that this is a setting for autoscaling. Something like:
```
The miniumum number of nodes to be set in the cluster for autoscaling. Must be 1 or greater ...
```